conversation.go

```diff
@@ -690,3 +690,53 @@ func (api *Client) MarkConversationContext(ctx context.Context, channel, ts stri
 	}
 	return response.Err()
 }
+
+type InviteSharedParams struct {
+	// Emails are the emails to be invited to the Slack Connect
+	Emails []string
+	// ExternalLimited determines whether the invitee is an external limited member.
+	// External limited members cannot invite other cannot invite others, export channel history,
+	// change channel visibility, topic, or name, and the channel will be set as private in the guest workspace.
+	// Defaults to true
+	ExternalLimited *bool
+	// UserIDs are the Slack IDs to be invited to the Slack Connect
+	UserIDs []string
+}
+
+var InvalidSharedInviteParamsError = errors.New("one of Emails or UserIDs must be supplied to InviteSharedParams")
+
+// InviteSharedContext sends array(s) of user IDs and/or emails an invitation to a Slack Connect channel with a custom context
+// At least one array of either emails or user IDs must be supplied to invite the recipient
+func (api *Client) InviteSharedContext(ctx context.Context, channelID string, params InviteSharedParams) error {
+	if len(params.UserIDs) < 1 && len(params.Emails) < 1 {
+		return InvalidSharedInviteParamsError
+	}
+
+	values := url.Values{
+		"channel": {channelID},
+	}
+	if params.Emails != nil {
```

Comment:
hmm the API docs are confusing, it says

> Optional email to receive this invite. Either emails or user_ids must be provided. Only one email or one user ID may be invited at a time.

but the type of the arguments is `array`

https://api.slack.com/methods/conversations.inviteShared

Reply:
Oh true I ran into this lol in our code it only goes one at a time. Gonna leave as is but the call side already loops and calls one by one right now.